packages/shared/src/types/feature-flags/feature-flags.ts

```diff
@@ -10,11 +10,10 @@ export enum FeatureFlagsKeysEnum {
   IS_QUOTA_LIMITING_ENABLED = 'IS_QUOTA_LIMITING_ENABLED',
   IS_EVENT_QUOTA_LIMITING_ENABLED = 'IS_EVENT_QUOTA_LIMITING_ENABLED',
   IS_TEAM_MEMBER_INVITE_NUDGE_ENABLED = 'IS_TEAM_MEMBER_INVITE_NUDGE_ENABLED',
-  IS_V2_EXPERIENCE_ENABLED = 'IS_V2_EXPERIENCE_ENABLED',
+  IS_V2_ENABLED = 'IS_V2_ENABLED',
```

Comment:
Won't we have change this at another places where that flag is being used like `apps/web/src/components/nav/RootNavMenu.tsx` line 50.